receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -463,15 +463,13 @@ func (jr *jReceiver) startCollector(host component.Host) error {
 		api_v2.RegisterCollectorServiceServer(jr.grpc, jr)
 
 		// init and register sampling strategy store
-		if len(jr.config.RemoteSamplingStrategyFile) != 0 {
-			ss, gerr := staticStrategyStore.NewStrategyStore(staticStrategyStore.Options{
-				StrategiesFile: jr.config.RemoteSamplingStrategyFile,
-			}, jr.logger)
-			if gerr != nil {
-				return fmt.Errorf("failed to create collector strategy store: %v", gerr)
-			}
-			api_v2.RegisterSamplingManagerServer(jr.grpc, collectorSampling.NewGRPCHandler(ss))
+		ss, gerr := staticStrategyStore.NewStrategyStore(staticStrategyStore.Options{
```

Comment:
https://github.com/jaegertracing/jaeger/blob/master/plugin/sampling/strategystore/static/strategy_store.go#L61

As you can see the strategy does not fail if the sampling file is missing. Is the service is not found the default is being used.